project/src/molecules/constructor.py

```diff
@@ -0,0 +1,33 @@
+"""Methods used to create systems from job statepoint."""
+import mbuild as mb
+import signac
+from mbuild import Box
+
+from project.src import molecules
+
+from .methane_ua import MethaneUA
+
+
+def SystemBuilder(job):
+    """Construct systems according to job statepoint."""
+    # Update this dict as new reicpes are made
+    molecule_dict = {
+        "methaneUA": MethaneUA,
+        "pentaneUA": None,
+        "benzeneUA": None,
+        "waterSPC/E": None,
+        "ethanolAA": None,
+    }
+    molecule = molecule_dict[job.sp["molecule"]]
+    vap_box = mb.Box()
+    liq_box = mb.Box()
+
+    vap_box_cpd = mb.fill_box(
+        compound=molecule, n_compound=job.sp["n_compounds"], box=vap_box
+    )
+    liq_box_cpd = mb.fill_box(
+        compound=molecule, n_compound=job.sp["n_compounds"], box=liq_box
+    )
+
+    system = mb.Compound([vap_box_cpd, vap_box_cpd])
+    return system
```

Comment:
Should we apply the forcefield before returning the system as well?

Reply:
I think this should return a mbuild compound, and the atomtyping step should happen in the individual `project.py`. I don't want to return a parmed structure here since some of our engines is requiring non-parmed writer.

Reply:
I think that should be a separate step, since we can serialize mbuild compounds to Json now. I have been burnt in the past where I have the initialize system including all these steps, but when FF application for more complicated systems can take a bit of time. I’d prefer to have the compound to inspect before we attempt to parametrize. We can make that a separate step that depends on the compound.Json files if we want. But I’m open for other suggestions.